basiclti/basiclti-portlet/src/webapp/WEB-INF/sakai/IMSBLTIPortlet.xml

```diff
@@ -1413,5 +1413,55 @@
           <configuration name="final.contentlink" value="true"/>
         </tool>
 
+        <tool
+            id="nyu.piazza"
+            title="Piazza"
+            description="Piazza">
+          <configuration name="sakai:portlet-pre-render" value="true" />
+          <category name="course" />
+          <category name="project" />
+          <category name="portfolio" />
+          <configuration name="functions.require" />
+          <configuration name="sakai:menuClass" value="icon-sakai-basiclti"/>
+          <configuration name="imsti.launch" />
+          <configuration name="imsti.xml" />
+          <configuration name="imsti.secret" />
+          <configuration name="imsti.key" />
+          <configuration name="imsti.pagetitle" value="Piazza"/>
+          <configuration name="imsti.tooltitle" value="Piazza"/>
+          <configuration name="imsti.newpage" value="on" />
+          <configuration name="imsti.maximize" value="on" />
+          <configuration name="imsti.frameheight" value="1024" />
+          <configuration name="imsti.debug" value="true" />
+          <configuration name="imsti.releasename" value="on"/>
+          <configuration name="imsti.releaseemail" value="on"/>
+          <configuration name="imsti.custom" />
+          <configuration name="imsti.allowsettings" value="off"/>
+          <configuration name="imsti.allowroster" value="on"/>
```

Comment:
Do we know if Piazza needs the course roster?   If not, we should turn this off.